modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -35,6 +40,16 @@ def get_appointments(start_date:, end_date:, include_pending:, pagination_params
 
         private
 
+        def extract_station_and_ien(appointment)
+          return nil if appointment[:identifier].nil?
+
+          identifier = appointment[:identifier].find { |id| id[:system]&.include? 'VistADefinedTerms/409_84' }
```

Comment:
It looks like the vaos logic this was borrowed from at `modules/vaos/app/services/vaos/v2/appointments_service.rb` was also looking for 409_85. was that left out intentionally?

Reply:
Nope. It looks like they just added that last week. I updated the logic